challenges/objects-arrays.js

```diff
@@ -88,33 +103,46 @@ zooAnimals = [{"animal_name":"Jackal, asiatic","population":5,"scientific_name":
 The zoo wants to display both the scientific name and the animal name in front of the habitats.  Return an array with only the animal and scientific names in it.  The individual values in the array should look like this "Name: Jackal, asiatic, Scientific: Canis aureus."
 
 */
-const animalNames = [];
+// const animalNames = animalNames.call( zooAnimals => {
+//   [`Name:${animal_name} , Scientific: ${scientific_name}`];
+// });
+const animalNames = []
 console.log(animalNames);
 
 /* Request 2: .map()    
 
 The zoos need a list of all their animal's names (names only, not scientific) converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
 
 */
-
-const lowerCase = [];
-console.log(lowerCase); 
-
+  const aniNames = [
```

Comment:
You are requied to use loop or any other means or tools that Javascript has at its disposal, not typing it manually by hand.